esp-hal/src/gpio/mod.rs

```diff
@@ -1917,6 +1973,12 @@ where
         self.pin.is_interrupt_set()
     }
 
+    /// Sets an interrupt handler.
+    #[inline]
+    pub fn set_interrupt_handler(&mut self, handler: fn()) {
```

Comment:
This should be of type `InterruptHandler`, but I suppose it can't because the prio would change it for all other pins.

Reply:
Well, this is where the annoying emulation pains begin: we can't use separate priority levels, so we either overwrite or ignore. I chose the option that doesn't need us to choose.